htdocs/core/lib/pdf.lib.php

```diff
@@ -1149,6 +1154,8 @@ function pdf_pagefoot(&$pdf, $outputlangs, $paramfreetext, $fromcompany, $marge_
 		}
 	}
 
+	$pdf->SetAutoPageBreak(0, 0); // Disable auto pagebreak
```

Comment:
If you disable pagebreak here, all content output after that are too long (for example the free text or content from pdf_pagefoot hook) may not generate the required pagebreak. Right ?

Reply:
Yes, pagebreak is disable for hook and default.

Reply:
So it means we introduce a bug when PDF_ALLOW_HTML_FOR_FREE_TEXT has a high content ?

Reply:
Personally, when my free text or custom footer is too high. I modify the values of MAIN_PDF_FREETEXT_HEIGHT and MAIN_PDF_MARGIN_BOTTOM.

If you want, I can add an option : PDF_FOOTER_DISABLE_PAGEBREAK ?

Reply:
Hello @eldy,
What do you think of my changes ?
Thank